src/Message/RefundRequest.php

```diff
@@ -128,17 +128,6 @@ public function getData()
 
     public function getEndpoint()
     {
-        $endPoint = $this->endpoint.'/charges/'.$this->getTransactionReference().'/refund';
-        $expandParams = $this->getExpand();
-        if ($expandParams && is_array($expandParams)) {
-            $endPoint = $endPoint . '?';
-            foreach ($expandParams as $idx => $param) {
-                $endPoint .= "expand[]=" . urlencode($param);
-                if ($idx !== count($expandParams) - 1) {
-                    $endPoint .= "&";
-                }
-            }
-        }
-        return $endPoint;
```

Comment:
Was trying to do this on Saturday. This doen't actually work, a separate field needs to be expanded for refunds (`balance_transaction` of the `refunds` in the `charge`). Decided to do this in a separate PR since this was already in ready to deploy